Makefile

```diff
@@ -61,12 +52,83 @@ BUILDFILE_AD_SERVER:=$(BUILDFILE_PREFIX).ad-server
 BUILDFILE_NIGHTLY_AD_SERVER:=$(BUILDFILE_PREFIX).nightly-ad-server
 BUILDFILE_CLIENT:=$(BUILDFILE_PREFIX).client
 BUILDFILE_TOOLBOX:=$(BUILDFILE_PREFIX).toolbox
+OS_PREFIX=$(addsuffix -,$(OS_NAME))
+TAG=$(OS_PREFIX)latest
+NIGHTLY_TAG=$(OS_PREFIX)nightly 
```

Comment:
Trailing whitespace towards the end of line.

Reply:
@anoopcs9  sorry, I currently don't see that context in my local copy ...

Reply:
```suggestion
NIGHTLY_TAG=$(OS_PREFIX)nightly
```